### PR TITLE
Add optional alt parameter to links.

### PR DESCRIPTION
**Pull request by Shinsina**
Screen Shot 2021-06-07 at 12 08 37 PM

### CODE REVIEW

**Comment by Shinsina** [comment]
[Add option for alt text on Email-X banner ads](https://trello.com/c/gcxrc7RQ/11-add-option-for-alt-text-on-email-x-banner-ads)

**Comment by brandonbk** [comment]
@Shinsina - closing this PR, since the alt property needs to be moved to the image where it is already supported.

File: packages/marko-core/components/elements/link.marko [hunk 1]
```diff
@@ -15,6 +15,7 @@ $ const rel = rels.length ? rels.join(" ") : null;
     title=input.title
     target=target
     class=classNames
+    alt=(input.alt ? input.alt : '' )
```
Comment (zarathustra323) [inline packages/marko-core/components/elements/link.marko:18]: You can simply do `alt=input.alt` here

**Review by zarathustra323 — CHANGES_REQUESTED** [review]

**Comment by Shinsina** [comment]
> @Shinsina - closing this PR, since the alt property needs to be moved to the image where it is already supported.

My understanding from the ticket was to put it on the ```<a>``` tag, personally I would have put it on the image because that's where alt normally goes but I was told "This is the "alt" text on the ```<a>``` tag", so that's what I did.

https://trello.com/c/gcxrc7RQ/11-add-option-for-alt-text-on-email-x-banner-ads

**Pull request closed without merging**